content/blog/2022-02-28-february-22-community-gems.md

```diff
@@ -0,0 +1,181 @@
+you need. The reason this happens is because DVC will automatically track `-O`
+outputs with Git internally since it assumes that any outputs that are not
+explicitly part of your `.gitignore` file is part of the experiment state that
+needs to be tracked.
+
+You should be able to explicitly add the `-O` output file(s) to your
+`.gitignore` as a workaround in the meantime.
+
+### [Is there an easy way to visualize DVC experiment results without using the command line?](https://discord.com/channels/485586884165107732/485596304961962003/930150143259459644)
+
+Good question @LucZ[Mad]!
+
+If you bring those experiments into your regular Git workflow like, using
+`dvc exp branch` to create a branch for any experiment you want to share, you
+could use [DVC Studio](https://studio.iterative.ai/) to visualize them.
+
+We're working on support for viewing any pushed experiments in Studio right now
+so if there's anything you want to see, make sure to comment on and follow
+[this issue](https://github.com/iterative/studio-support/issues/45).
+
+### [Is it possible to change the CML runner shutdown to stopping the instance after the idle timeout instead of terminating the instance using self-hosted runners on AWS?](https://discord.com/channels/485586884165107732/728693131557732403/933674203796873226)
+
+This is another fantastic question from @jotsif!
+
+Unfortunately no, the instance will be destroyed. If you're trying to preserve
+the cache to try and speed up your experimentation time, you could
+[check this out](https://aws.amazon.com/premiumsupport/knowledge-center/s3-transfer-data-bucket-instance/)
+if you're using S3 for your remote storage.
+
+Just be cautious since an instance that is in the "off" state might still be
+considered in use for billing purposes. It's best to let the CML runner
+terminate your instance and run `dvc pull` to restore the data.
+
+### [Where can I find more details on how the DVC Studio free version differs from the enterprise version?](https://discord.com/channels/485586884165107732/841856466897469441/933324508570472497)
+
+Thanks for asking @Abdi!
+
+You can find more info about the different
+[DVC Studio tiers here](https://studio.iterative.ai/#pricing).
+
+The Free version has all the features most individual users need, like
+connecting to ML repositories, creating views, submiting experiments, and
+generating plots. The Teams version allows you to create large teams for better
+collaboration and sharing of views and settings with everyone. The Enterprise
+version is more for needs around compliance, dedicated support, and on-premise
+installation.
+
+If you are trying to decide which plan to select, please email us at
+`info@iterative.ai` and we'll help you figure it out based on your needs.
+
+### [How do you `dvc commit` or get the `dvc status` of each case in a `foreach` stage?](https://discord.com/channels/485586884165107732/563406153334128681/938649682492686366)
```

Comment:
```suggestion
### [How to `dvc commit` or get the `dvc status` of each stage in a `foreach` loop?](https://discord.com/channels/485586884165107732/563406153334128681/938649682492686366)
```

Reply:
This actually is not supported right now 😬 . See https://github.com/iterative/dvc/issues/7323.

Reply:
:facepalm:

Reply:
😬 Ah... Ok. I'll just remove this question. 🚶🏽‍♀️ Thanks for catching this @dberenbaum!